Online/Resource/WorldSession.cs

```diff
@@ -128,11 +135,18 @@ public override void ReadTo(OnlineResource resource)
                 if (resource.isActive)
                 {
                     var ws = (WorldSession)resource;
+
+                    ws.world.game.clock = clock;
```

Comment:
Confirm that the clock never regresses
```suggestion
                    ws.world.game.clock = Mathf.max(clock, ws.world.game.clock);
```

Reply:
@invalidunits does this affect devtools clock reset?